examples/tour.jl

```diff
@@ -167,9 +173,8 @@ transform(filter(ann -> within_signal(ann, sig), annotations),
 # requested set of `span`s heavily overlap, this approach may be less efficient than simply loading
 # the whole file upfront. Here we demonstrate the latter as an alternative (note: in the future, we
 # want to support an optimal batch loader):
-samples = load(sig)
-transform(filter(ann -> within_signal(ann, sig), annotations),
-          :span => ByRow(span -> view(samples, :, translate(span, -start(sig.span)))) => :samples)
+samples = load(grp[1, :])
+transform!(grp, :span => ByRow(s -> view(samples, :, translate(s, -start(s)))) => :samples)
```

Comment:
```suggestion
samples = load(first(annotated_signals))
transform(annotated_signals, :span => ByRow(s -> view(samples, :, translate(s, -start(s)))) => :samples)
```
This suggestion is paired with the previous